src/daemon/daemon.cpp

```diff
@@ -1680,6 +1680,26 @@ try // clang-format on
             delayed_shutdown_instances.erase(name);
         }
 
+        try
+        {
+            auto& sshfs_mounts = mount_threads.at(name);
+            for (const auto& sshfs_mount : sshfs_mounts)
+            {
+                mpl::log(mpl::Level::debug, "delete",
+                         fmt::format("Stopping mount '{}' in instance \"{}\"", sshfs_mount.first, name));
+                sshfs_mount.second->stop();
+            }
+        }
+        catch (const std::out_of_range&)
+        {
+            mpl::log(mpl::Level::debug, "delete", fmt::format("No mounts to stop for instance \"{}\"", name));
+        }
+        catch (const std::exception& e)
+        {
+            mpl::log(mpl::Level::error, "delete",
+                     fmt::format("Error stopping mount(s) in instance \"{}\": {}", name, e.what()));
+        }
```

Comment:
If the catch is maintained for logging, shouldn't the exception be re-thrown? `std::exception` is very general and it could signal an important problem (an unrecoverable one, in the worst case). I am not sure we'd want to claim success to the client with only a message in the log in that case.

Reply:
The re-throw depends on if we want to fail the delete or not if an exception besides std::out_of_range is thrown during the mount stop.  I don't necessarily view that as a fatal error since the instance is going to be shut down and "deleted" any ways.

Reply:
This block as been removed altogether, so no need to re-throw:)